### PR TITLE
add required & other option

### PR DESCRIPTION
**Pull request by kelimuttu**

### CODE REVIEW

File: assets/js/impactform.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 $('#input-feedback').hide();
 
+var today = new Date();
 $('[data-toggle="datepicker"]').datepicker({
     todayHighlight: true,
+    endDate: today,
```
Comment (MichaelKohler) [inline assets/js/impactform.js]: nice catch!

**Comment by MichaelKohler** [comment]
Works like a charm, just one minor thing, when I don't fill out the name and email address, the input fields aren't marked as red, even though they are marked as "required". Could you have a quick look into that? Apart from that it works like a charm! Thanks so much!

screen shot 2017-05-21 at 16 56 41

**Comment by kelimuttu** [comment]
A tooltip will show up when the input form is not filled out. In your case above, it's not come out because the warning only popped up on the first unanswered question.

**Comment by MichaelKohler** [comment]
This seems stupid. But yeah, let's file a Firefox patch :P I'm merging..

**Pull request merged**